ui/app/components/actions-flyout.hbs

```diff
@@ -0,0 +1,36 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+{{#if this.nomadActions.flyoutActive}}
+  <Hds::Flyout id="actions-flyout"
+    @onClose={{this.nomadActions.closeFlyout}} @size="large" as |Fly|
+  >
+    <Fly.Header>
+      <h3>
+        Actions
+      </h3>
+      {{#if this.contextualActions.length}}
+        <ActionsDropdown
+          @actions={{this.contextualActions}}
+          @allocation={{this.allocation}}
+          @context={{if this.task this.task this.job}}
+        />
+      {{/if}}
+      {{#if this.nomadActions.runningActions.length}}
+        <Hds::Button @text="Stop All" @color="critical" @size="medium" {{on "click" this.nomadActions.stopAll}} />
+      {{/if}}
+      {{#if this.nomadActions.finishedActions.length}}
+        <Hds::Button @text="Clear Finished Actions" @color="secondary" @size="medium" {{on "click" this.nomadActions.clearFinishedActions}} />
+      {{/if}}
+    </Fly.Header>
+    <Fly.Body>
+      <ul class="actions-queue">
+        {{#each this.actionInstances as |instance|}}
+          <ActionCard @instance={{instance}} />
+        {{/each}}
```

Comment:
TODO: Because we can clear this queue out ("Clear finished actions"), we can arrive in an "Actions flyout is open, but there are no actions" state. Handle this with explanatory text + docs link.